tools/doc/type-parser.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+const nodeDocUrl = '';
+const jsDocUrl = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/' +
+  'Reference/Global_Objects/';
+const jsGlobalTypes = [
+  'Error', 'Number', 'Object', 'Boolean', 'String', 'Function', 'Symbol',
+  'Array', 'Uint8Array', 'Uint16Array', 'Uint32Array', 'Int8Array',
+  'Int16Array', 'Int32Array', 'Uint8ClampedArray', 'Float32Array',
+  'Float64Array', 'Date', 'RegExp', 'ArrayBuffer', 'DataView', 'Promise',
+  'Null'
+];
+const typeMap = {
+  'Buffer': 'buffer.html#buffer_class_buffer',
+  'Handle': 'net.html#net_server_listen_handle_callback',
```

Comment:
Handle probably needs to be more specific, there are many types of handles, although they all inherit from AsyncWrap afaik.